shm/Player.cpp

```diff
@@ -22,8 +21,8 @@ size_t Player::countAvailableSpace() {
     auto occupiedSpace = std::accumulate(begin(ship_->getAllCargos()),
                                          end(ship_->getAllCargos()),
                                          0,
-                                         [](const Cargo& cargoFirst, const Cargo& cargoSecond) {
-                                             return cargoFirst.getAmount() + cargoSecond.getAmount();
+                                         [](int accumulated, const auto cargoSecond) {
```

Comment:
Maybe we should now change the name of the second argument to something more relevant?

Reply:
Changed